lib/cryptoexchange/exchanges/saturn_network/services/market.rb

```diff
@@ -0,0 +1,50 @@
+module Cryptoexchange::Exchanges
+  module SaturnNetwork
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+
+        def fetch
+          output = super ticker_url
+          adapt_all(output)
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::SaturnNetwork::Market::API_URL}/returnTicker.json"
+        end
+
+        def adapt_all(output)
+          output.map do |pair|
+            base = pair[0].split('_').first
+            target = pair[1]["symbol"]
+            market_pair = Cryptoexchange::Models::MarketPair.new(
+              base: base,
+              target: target,
+              market: SaturnNetwork::Market::NAME
+            )
+            adapt(pair, market_pair)
+          end
+        end
+
+        def adapt(output, market_pair)
+          ticker = Cryptoexchange::Models::Ticker.new
+
+          ticker.base      = market_pair.base
+          ticker.target    = market_pair.target
+          ticker.market    = SaturnNetwork::Market::NAME
+          ticker.last      = NumericHelper.to_d(output[1]["last"])
+          ticker.high      = NumericHelper.to_d(output[1]["highestBid"])
+          ticker.low       = NumericHelper.to_d(output[1]["lowestAsk"])
+          ticker.volume    = NumericHelper.to_d(output[1]["baseVolume"])
```

Comment:
```

name | "Saturn Classic DAO Token"
-- | --
symbol | "SATURN"
quoteVolume | "30000"
baseVolume | "1.45"
highestBid | "0.00003"
lowestAsk | "0.00005"
last | "0.00005"
percentChange | "0.06230889172387269"
```

From the example response. We want the volume to be in base in our convention.
If you fix the base/target by reversing it (https://github.com/coingecko/cryptoexchange/pull/1361/files#r255827784), then you will want to take quoteVolume

Reply:
You need to reverse it because otherwise it would be ETH/SATURN
And by assigning `last` it would be deem as 0.0003 SATURN per ETH